organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
+open society.
+
+**Open Society**: An open society promotes freedom of thought and expression,
+personal responsibility, individualism, cooperation, and equality of all
+citizens.
+
+Code of Conduct
+===============
+
+It is expected that members behave in a way that allows Hacker Society to
+achieve its mission.  Participants in Hacker Society events are expected to
+treat each other with dignity and respect for each other's perspectives,
+experiences, and knowledge.  In particular, participants should recognize that
+people of all skill levels are welcome, and should always be willing to help
+others learn.
+
+To support Hacker Society's mission of: learning technology from each other,
+building community, and promoting equality, Hacker Society must be a place where
+people can be comfortable regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, age, or
+religion. Therefore, we do not tolerate:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
```

Comment:
> Although on the other hand, significant technical (or other) contributions should not excuse unacceptable behaviour.

That is not what is being said here. This code as written does not do that. What I said above does not do that. I am not doing that.

This attitude, "contribution is no excuse for bad behavior," is what this entire paragraph is about. People may have said something that a majority feels is politically incorrect outside of hacker society. Should that person be allowed to participate in hacker society? yes! As long as they don't violate the specific restrictions in place for hacker society.

It is when some one says or does something at hacker society that this policy comes into force. It is not about excluding people for behavior or statements outside of hacker society. For instance, we have had talks on http://urbit.org before by people who have interned there. Now it so happens that Curtis Yarvin the founder of urbit has an only persona unrelated to urbit or technology that some people find offensive. We should not bar Curtis from speaking on Urbit and on technology as long as he obeys the code of conduct and guidelines for speaking at hacker society.

This code of conduct is not about trying to ensure all hacker society participants morally or ethically upstanding. It is trying to ensure that:

1. People can learn about technology from each other
2. We can have a functioning respectful community
3. We can promote equality among the participants
4. We can promote freedom of thought and expression
5. We can foster an open society.

We can only do those things if:

1. We have a welcoming environment
2. We have a non-sexualized environment
3. A free environment where people can explore ideas